python/LLM Examples/PAI-Cohere.ipynb

```diff
@@ -262,7 +265,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.3"
+   "version": "3.10.9"
```

Comment:
not sure how to change this 🤷 I guess it's just metadata about what version I'm using

Reply:
nvm got it sorted